docs/en/sql-reference/functions/type-conversion-functions.md

```diff
@@ -1426,33 +1426,29 @@ dateTimeToSnowflake(value)
 
 -   `value` — Date and time. [DateTime](../../sql-reference/data-types/datetime.md).
 
-
 **Returned value**
 
--   `value` converted to the `Int64` data type as the first snowflake id at that time.
+-   Input value converted to the [Int64](../data-types/int-uint.md) data type as the first Snowflake ID at that time.
```

Comment:
`first Snowflake ID` sounds unclear, but I don't know how to make it better.

Reply:
@alesapin 
May be, `the least Snowflake ID at that time`?